R/parents.R

```diff
@@ -0,0 +1,137 @@
+#' Getter and setter for specific parent
+#'
+#' @param join_keys_obj (`join_keys`) object to retrieve.
+#' @param dataset_name (`character(1)`)
+#'
+#' @export
+#'
+#' @examples
+#' jk <- join_keys()
+#' parent(jk, "ds1")
+#' parents(jk) <- list("ds2" = "ds3")
+#' parent(jk, "ds2")
+parent <- function(join_keys_obj, dataset_name) {
+  if (missing(dataset_name)) {
+    return(NULL)
```

Comment:
Shouldn't this be an error?

Reply:
I think so as well

Reply:
Sure thing :+1: I see now that a lot of [legacy logic](https://github.com/insightsengineering/teal.data/blob/9aae01c613bb889eed0a6898faf93e99d42ba0ef/R/JoinKeys.R#L198) should've been re-written

Reply:
Done in 1d40996f